### PR TITLE
fix message in modal window

### PR DESCRIPTION
**Pull request by ostgit12**
## Description
fix messages in dialog windows

#### Screenshots
before fix
beforeFixMessageInModalWindow

after fix
afterFixMessageInModalWindow

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/configs/messages.js [hunk 1]
```diff
@@ -16,9 +14,6 @@ const messages = {
   PATTERN_REMOVE_MESSAGE: 'Ви впевнені, що хочете видалити цей гобелен?',
   DELETE_SIZE_MESSAGE: 'Ви впевнені, що хочете видалити цей розмір?',
   NO_SIZES_MESSAGE: 'Розміри відсутні',
-  NO_CATEGORY_MESSAGE: 'Категорії відсутні',
```
Comment (grebiniuk) [inline src/configs/messages.js:19]: Why these confs were removed?
Rows 19-21

File: src/configs/messages.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
 const messages = {
-  SAVE_CHANGES: 'Зберегти зміни',
```
Comment (grebiniuk) [inline src/configs/messages.js:2]: Why these confs were removed?
Rows 2-3

File: src/configs/button-titles.js [hunk 3]
```diff
@@ -63,16 +63,8 @@ const buttonTitles = {
   EVENT_TITLE: 'Подія',
   SEND_CONFIRM: 'Надіслати підтвердження',
   SEND_CODE: 'Надіслати код для підтвердження',
-  CONFIRM: 'Так',
   EXIT_WITHOUT_SAVING: 'Вийти без збереження',
-  STATUS: 'Статус замовлення',
```
Comment (grebiniuk) [inline src/configs/button-titles.js:68]: Why these confs were removed?
Rows 68-75

**Review by grebiniuk — CHANGES_REQUESTED** [review]
Please answer my comments

**Review by grebiniuk — CHANGES_REQUESTED** [review]
Added questions in the last commit

**Review by grebiniuk — CHANGES_REQUESTED** [review]
There are still some questions without the answer/fix

**Review by grebiniuk — APPROVED** [review]

**Review by yuliiakucher — APPROVED** [review]

File: src/configs/messages.js (hunk 1 above)
Comment (ostgit12) [reply src/configs/messages.js]: this issue is fixed successfully

Comment (ostgit12) [reply src/configs/messages.js]: this issue is fixed successfully

Comment (ostgit12) [reply src/configs/messages.js]: this issue is fixed successfully

Comment (ostgit12) [reply src/configs/messages.js]: this issue is fixed successfully

**Pull request merged**